client/src/css/Map.css

```diff
@@ -0,0 +1,5 @@
+.mapContainer {
+    padding-top: 3vh;
+    text-align: center;
+    height: 100vh;
+}
```

Comment:
We should keep all CSS for a given component in the same place, ideally in `withStyles`